pkgs/development/python-modules/stups-cli-support/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, clickclick
+, dnspython
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "stups-cli-support";
+  version = "1.1.20";
+  doCheck = false;
+  propagatedBuildInputs = [
+    clicklick
+    dnspython
+    requests
+  ];
+  src = super.fetchPypi {
```

looks like you pulled this out of an overlay? :)
```suggestion
  src = fetchPypi {
```